org/mozilla/jss/nss/PR.c

```diff
@@ -106,7 +106,7 @@ Java_org_mozilla_jss_nss_PR_Close(JNIEnv *env, jclass clazz, jobject fd)
     }
 
     PRStatus ret = PR_Close(real_fd);
-    if (ret == PR_SUCCESS) {
+    if (ret == PR_SUCCESS && clear == JNI_TRUE) {
```

Comment:
It looks like this condition only happens in JSS tests. If there is no real usage where `clear == JNI_TRUE` we might as well simplify the code to no longer call `JSS_clearPtrFromProxy()`. See also my other comment about not calling `JSS_clearPtrFromProxy()`.

Reply:
Yeah I think we could update this in a separate PR and simplify how these are handled.